pkgs/development/compilers/llvm/15/compiler-rt/default.nix

```diff
@@ -97,8 +97,6 @@ stdenv.mkDerivation {
     substituteInPlace cmake/builtin-config-ix.cmake \
       --replace 'set(X86 i386)' 'set(X86 i386 i486 i586 i686)'
   '' + lib.optionalString stdenv.isDarwin ''
-    substituteInPlace cmake/builtin-config-ix.cmake \
-      --replace 'set(ARM64 arm64 arm64e)' 'set(ARM64)'
```

Comment:
Please apply this change to llvm/git/compiler-rt too if needed, cc @rrbutani

Reply:
Looks like that's already correct on master, since #186575, so no action needed: https://github.com/NixOS/nixpkgs/blob/5649872ad1ebc27c8f653e53c4d20260008dbd68/pkgs/development/compilers/llvm/git/compiler-rt/default.nix#L98-L101

It was probably just a small confusion in all the merges around LLVM 15, which happened after #186575.

Reply:
Perfect!

Reply:
Apologies, this was another mistake I made; should have been included in https://github.com/NixOS/nixpkgs/pull/194634/commits/09b8886d5e7e2511a8e68cfe04f11cabb07da539.

Reply:
No apology necessary, and thank you for your hard work. LLVM upgrades seem like an enormous effort. There wouldn't be a Swift 5.8 upgrade without an LLVM 15 upgrade!